src/sentry/static/sentry/app/components/globalModal.jsx

```diff
@@ -38,6 +38,12 @@ class GlobalModal extends React.Component {
       options.onClose();
     }
 
+    // Focus this to get hotkeys to keep working
+    let containerEl = document.querySelector('.main-container');
+    if (containerEl) {
+      containerEl.focus();
```

Comment:
This seems hacky, but not sure of a better solution.

Reply:
Could it be done like in 
https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/components/assigneeSelector.jsx#L292 ?

Reply:
~~The problem is when you close the modal, global hot keys stop working because the main body doesn't have focus. This is focusing an app container that's higher up in the tree.~~ Ah after looking at it, `.main-container` is the parent so we may be able to just use a callback, for some reason I thought it was higher up in the tree.